app/models/benefitting_country.rb

```diff
@@ -7,6 +7,16 @@ def all
       @all ||= Codelist.new(type: "benefitting_countries", source: "beis").map { |country| new_from_hash(country) }
     end
 
+    def find_by_code(code)
+      all.find { |country| country.code == code }
+    end
+
+    def region_from_country_codes(codes)
+      codes.map { |c| find_by_code(c) }
+        .map { |c| c.regions }
+        .inject(:&).last || Region::DEVELOPING_COUNTRIES_REGION
```

Comment:
This is where I feel we could have an actual issue. if the `regions` array here is not in the expected order the logic we base this on could be broken, here we look for the last region that comes back from all of them and say 'no regions are the same across countries so it must be 998' - but they may not match because of an ordering problem?